modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/throttling/ThrottlingUtils.java

```diff
@@ -20,23 +20,6 @@
 
 import java.util.Calendar;
 
-/*
- *Copyright (c) 2024, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
- *
- *WSO2 Inc. licenses this file to you under the Apache License,
- *Version 2.0 (the "License"); you may not use this file except
- *in compliance with the License.
- *You may obtain a copy of the License at
- *
- *http://www.apache.org/licenses/LICENSE-2.0
- *
- *Unless required by applicable law or agreed to in writing,
- *software distributed under the License is distributed on an
- *"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- *KIND, either express or implied.  See the License for the
- *specific language governing permissions and limitations
- *under the License.
- */
```

Comment:
Is there a reason that we remove the license header from the file?

Reply:
license header has added twice. Removed one.

Reply:
Thanks for the clarification. +1 to proceed.